assets/queries/openAPI/2.0/operation_object_parameters_with_body_and_formatdata/test/negative1.json

```diff
@@ -0,0 +1,51 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Simple API Overview",
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/": {
+      "get": {
+        "operationId": "listVersionsv2",
+        "summary": "List API versions",
+        "responses": {
+          "200": {
+            "description": "200 response"
+          }
+        },
+        "parameters": [
+          {
+            "name": "limit2",
+            "in": "body",
+            "description": "max records to return",
+            "required": true,
+            "schema": {
+              "type": "object"
+            }
+          },
+          {
+            "name": "minlimit",
+            "in": "body",
+            "description": "min records to return",
+            "required": true,
+            "schema": {
+              "type": "object"
+            }
+          }
+        ]
+      }
+    }
+  },
+  "parameters": {
+    "limitParam": {
+      "name": "limit",
+      "in": "formatData",
+      "description": "max records to return",
+      "required": true,
+      "schema": {
```

Comment:
`schema` is not a valid value when `"in": "formatData"`. It is only a valid value when `"in": "body"`. The same happens in the other samples